src/main/java/mekanism/common/integration/OreDictManager.java

```diff
@@ -407,33 +415,21 @@ public static void init(RegistryEvent.Register<IRecipe> event) {
         }
 
         for (ItemStack coal : OreDictionary.getOres("blockCoal", false)) {
+            coal = StackUtils.size(coal, 1);
             RecipeHandler.addPRCRecipe(coal, new FluidStack(FluidRegistry.WATER, 1000), new GasStack(MekanismFluids.Oxygen, 1000),
                   new ItemStack(MekanismItems.OtherDust, 9, 3), new GasStack(MekanismFluids.Hydrogen, 1000), 0, 900);
         }
 
         for (ItemStack coal : OreDictionary.getOres("blockCharcoal", false)) {
+            coal = StackUtils.size(coal, 1);
             RecipeHandler.addPRCRecipe(coal, new FluidStack(FluidRegistry.WATER, 1000), new GasStack(MekanismFluids.Oxygen, 1000),
                   new ItemStack(MekanismItems.OtherDust, 9, 3), new GasStack(MekanismFluids.Hydrogen, 1000), 0, 900);
         }
 
-        for (ItemStack coal : OreDictionary.getOres("dustCoal", false)) {
-            RecipeHandler.addPRCRecipe(coal, new FluidStack(FluidRegistry.WATER, 100), new GasStack(MekanismFluids.Oxygen, 100),
-                  new ItemStack(MekanismItems.OtherDust, 1, 3), new GasStack(MekanismFluids.Hydrogen, 100), 0, 100);
-        }
-
-        for (ItemStack coal : OreDictionary.getOres("dustCharcoal", false)) {
-            RecipeHandler.addPRCRecipe(coal, new FluidStack(FluidRegistry.WATER, 100), new GasStack(MekanismFluids.Oxygen, 100),
-                  new ItemStack(MekanismItems.OtherDust, 1, 3), new GasStack(MekanismFluids.Hydrogen, 100), 0, 100);
-        }
-
-        for (ItemStack sulfur : OreDictionary.getOres("sulphur", false)) {
-            RecipeHandler.addEnrichmentChamberRecipe(sulfur, new ItemStack(Items.GUNPOWDER));
-        }
-
         for (ItemStack sawdust : OreDictionary.getOres("dustWood", false)) {
-            ItemStack dust = StackUtils.size(sawdust, 8);
+            sawdust = StackUtils.size(sawdust, 8);
             //TODO: 1.14 evaluate adding a charcoal dust item to Mekanism, and if so use that instead of charcoal here
-            RecipeHandler.addEnrichmentChamberRecipe(dust, new ItemStack(Items.COAL, 1, 1));
+            RecipeHandler.addEnrichmentChamberRecipe(sawdust, new ItemStack(Items.COAL, 1, 1));
             RecipeHandler.addPRCRecipe(sawdust, new FluidStack(FluidRegistry.WATER, 20), new GasStack(MekanismFluids.Oxygen, 20),
```

Comment:
This is changing this line from using 1 sawdust to using 8 right?

Reply:
no. you are right i wasn't thinking about the bottom line :)

Reply:
ill fix that rn

Reply:
ok fixed that.